server/src/test/java/com/ca/lsp/cobol/usecases/TestExecCicsContainerStatementsArgumentsOrder.java

```diff
@@ -51,7 +51,7 @@
           + "       {#*MAINLINE}.\n"
           + "           exec cics put container({$xml-cont})\n"
           + "                         channel({$outbound-channel})\n"
-          + "                         from(DFH-BODY(15:{$Scratch-len}))\n"
+          + "                         from({$DFH-BODY}(15:{$Scratch-len}))\n"
```

Comment:
Now I realize that it is not required to tag each significant token into tests.
For example, in `TestOutlineTree` I have no tags at all and it is OK. Must we check that all needed tokens marked or not?